app/_how-tos/get-started-with-metering-and-billing.md

```diff
@@ -0,0 +1,235 @@
+
+In this guide, you'll enable API Gateway requests for metering. This will meter API request traffic in Metering & Billing so that you can charge customers for API traffic usage.
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
+1. Enable **API Gateway Requests**.
+
+## Create a feature
+
+Meters collect raw usage data, but features make that data billable. Without a feature, usage is tracked but not invoiced. Now that you're metering API consumption, you need to associate traffic from the `example-service` Gateway Service with a feature as something you want to price or govern. 
+
+Features are customer-facing, and show up on the invoice for paid plans. Feature examples could include things like flight data requests, GPT-5 input tokens, or available LLM models. 
+
+In this guide, you'll create a feature for the `example-service` you created in the prerequisites.
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
+1. In the Metering & Billing sidebar, click **Product Catalog**.
+1. Click **Create Feature**.
+1. In the **Name** field, enter `example-service`.
+1. From the **Meter** dropdown menu, select "kong_konnect_api_request". 
+1. Click **Add group by filter**. 
+   The group by filter ensures you only bill for traffic to `example-service`, not all {{site.base_gateway}} traffic. This lets you offer different pricing for different APIs.
+1. From the **Group by** dropdown menu, select "service_name".
+1. From the **Operator** dropdown menu, select "Equals".
+1. From the **Value** dropdown menu, select "example-service".
+1. Click **Save**. 
+
+## Create a Premium plan
+
+Plans are the core building blocks of your product catalog. They are a collection of rate cards that define the price and access of a feature. Plans can be assigned to customers by starting a subscription.
+
+A rate card describes price and usage limits or access control for a feature or item. Rate cards are made up of the associated feature, price, and optional usage limits or access control for the feature, called entitlements.
+
+In this section, you'll create a Premium plan that grants paying customers access to the `example-service` at a rate of 5,000 requests per month:
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
+1. In the Metering & Billing sidebar, click **Product Catalog**.
+1. Click the **Plans** tab.
+1. Click **Create Plan**.
+1. In the **Name** field, enter `Premium`.
+1. In the **Billing cadence** dropdown menu, select "1 month".
+1. Click **Save**.
+1. Click **Add Rate Card**.
+1. From the **Feature** dropdown menu, select "example-service".
+1. Click **Next**.
+1. From the **Pricing model** dropdown menu, select "Usage Based".
+1. In the **Price per package** field, enter `1`.
+1. In the **Quantity per package** field, enter `5000`.
```

Comment:
Usage-based price doesn't have a quantity. Only the packaged price has it, but we changed it to UBP.